### PR TITLE
Load Couchdb

### PR DESCRIPTION
**Pull request by krisdo**
### Why

Need to seed CouchDB with fake data

### How

Create fake data with helper functions and create a function to seed the database 

### What

- Copied and modified data generator functions for CouchDB.
- Installed NPM package, couchimport, to parse csv and bulk write into couchdb
- Implemented a function to seed 3 CSV files into database
- Refactored code to add headers to CSV files

### Where

Data Generating Helper Functions
- ./database/data-generator/generate-gen-discounts-couch.js
- ./database/data-generator/generate-pub-discounts-couch.js
- ./database/data-generator/generate-products-couch.js

CouchDB connection
- ./database/database-couchdb/index.js

Parse and Write CSV files to CouchDB
-./database/database-couchdb/seed.js

### TODO

- Add logic to check couchdb if the database exist
- Write query functions to interact with the db

### Notes

Please ignore these files.
- price_and_promotion/database/database-postgres/query.sql
- price_and_promotion/database/database-postgres/importData.js
- price_and_promotion/database/database-postgres/db.js
- price_and_promotion/database/database-couchdb/views.js

alt-text

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -2,8 +2,11 @@ PORT=3006
 DB_NAME='127.0.0.1:27017/PriceAndPromo'
 SC_ATTR="PAP-styles"
 CLIENT=pg
-HOST=127.0.0.1
-PG_PORT=5432
-PG_USER=student
-PG_PASSWORD=student
-DATABASE=priceandpromotions
+PGHOST=127.0.0.1
```
Comment (cywang117) [inline .env]: Should this be .gitignore'd ? :0

File: database/data-generator/generate-pub-discounts-couch.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+const faker = require('faker');
```
Comment (cywang117) [inline database/data-generator/generate-pub-discounts-couch.js:1]: Your data gen functions all look good! Again, liking that modularity.

File: database/database-postgres/importData.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// Import required modules
```
Comment (cywang117) [inline database/database-postgres/importData.js]: Ignoring!

**Review by cywang117 — APPROVED** [review]
Looks like you went _ape_ with the refactors - very straightforward, and no issues. Appreciate the comment telling me which files to look at, also liking the comments surrounding your code 🌝 

Good job!

File: .env (hunk 1 above)
Comment (krisdo) [reply .env]: hey! that's strange. It's always been in my .gitingore. I will check it out.

**Pull request merged**